composer.json

```diff
@@ -33,8 +33,11 @@
         "behat/symfony2-extension": "^2.1.1",
         "behatch/contexts": "3.1.0",
         "doctrine/annotations": "^1.2",
+        "doctrine/data-fixtures": "^1.2.2",
```

Comment:
Temporary workaround for https://github.com/doctrine/DoctrineMongoDBBundle/issues/557